pkg/resources/runtime/kernel_params_spec.go

```diff
@@ -0,0 +1,79 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package runtime
+
+import (
+	"fmt"
+
+	"github.com/cosi-project/runtime/pkg/resource"
+	"github.com/cosi-project/runtime/pkg/resource/meta"
+)
+
+// NamespaceName contains configuration resources.
+const NamespaceName resource.Namespace = "runtime"
```

Comment:
this one is already defined in `v1alpha1/` as a `NamespaceName`, should we merge these two together? we can make this reference `v1alpha1.NamespaceName` for now, and we can merge two folders later on

Reply:
will make a ref